Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN dotnet restore "utils/actions/actions.csproj" -a $TARGETARCH
 WORKDIR "/actions"
 RUN dotnet build "utils/actions/actions.csproj" -c Release -a $TARGETARCH -o /actions/build
 
-FROM --platform=$BUILDPLATFORM btcpayserver/btcpayserver:1.13.5-altcoins
+FROM --platform=$BUILDPLATFORM btcpayserver/btcpayserver:2.0.0-altcoins
```

Comment:
A bug in btcpayserver:2.0.0-altcoins perhaps?

Reply:
No, i changed something in the build, working on it